CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 # CMakeLists.txt - CMake configuration file for EPANETMSX 1.1
 
-cmake_minimum_required(VERSION 3.9)
+cmake_minimum_required(VERSION 3.12)
 project(EPANETMSX LANGUAGES C)
 
 include_directories(
```

Comment:
@karrowood do these relative paths work in Win? I added some info to the compile notes about just having the epanet repo on a common root dir, so these include/link lines work ok on Mac/nix, but I would like to confirm support in Windows too.

Reply:
I just tried it with them both in the same root directory, and it did not work successfully it says that there is a CMAKE error because the "EPANET_LIB" variable is set to NOTFOUND.

Reply:
the epanet lib should be build and available in `../EPANET/build/lib`, e.g.:

```
cd EPANET/
mkdir build && cd build
cmake .. && make
```

Reply:
I made some changes to the CMakeLists.txt in order to have this working on windows, there was a problem since the directories built inside of a Release folder in windows rather than just directly in the build/lib folder. I think it should still work on Mac/*nix, but it is probably best that you make sure of that.